packages/core/src/mls/MLSService/stores/pendingProposalsStore/pendingProposalsStore.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CommonMLS} from '../../../types';
+
+export type PendingProposalsParams = {
+  firingDate: number;
+} & CommonMLS;
```

Comment:
Can you check if it still makes sense to have this `CommonMLS`? I feel like it's only used here (in which case we can inline the `groupId` in this type